tcases-lib/src/main/java/org/cornutum/tcases/util/XmlWriter.java

```diff
@@ -125,7 +128,7 @@ public void writeAttribute( String name, String value)
     print( " ");
     print( name);
     print( "=\"");
-    print( value);
+    print( StringEscapeUtils.escapeXml10(value));
```

Comment:
not sure about all the differences between XML 1.0 and 1.1. This one suggests to go with 1.0 when in doubt: https://stackoverflow.com/questions/6260975/should-i-learn-xml-1-0-or-xml-1-1